sdv/data_processing/numerical_formatter.py

```diff
@@ -0,0 +1,101 @@
+"""Formatter for numerical data."""
+import sys
+
+import numpy as np
+import pandas as pd
+
+MAX_DECIMALS = sys.float_info.dig - 1
+INTEGER_BOUNDS = {
+    'Int8': (-2**7, 2**7 - 1),
+    'Int16': (-2**15, 2**15 - 1),
+    'Int32': (-2**31, 2**31 - 1),
+    'Int64': (-2**63, 2**63 - 1),
+    'UInt8': (0, 2**8 - 1),
+    'UInt16': (0, 2**16 - 1),
+    'UInt32': (0, 2**32 - 1),
+    'UInt64': (0, 2**64 - 1),
+}
+
+
+class NumericalFormatter:
+    """Formatter for numerical data.
+
+    Args:
+        learn_rounding_scheme (bool):
+            Whether or not to learn what place to round to based on the data seen during ``fit``.
+            If ``True``, the data returned by ``reverse_transform`` will be rounded to that place.
+            Defaults to ``False``.
+        enforce_min_max_values (bool):
+            Whether or not to clip the data returned by ``reverse_transform`` to the min and
+            max values seen during ``fit``.
+            Defaults to ``False``.
+        representation (dtype):
+            Accepts ``'Int8'``, ``'Int16'``, ``'Int32'``, ``'Int64'``, ``'UInt8'``, ``'UInt16'``,
+            ``'UInt32'``, ``'UInt64'``, ``'Float'``.
+            Defaults to ``'Float'``.
+    """
+
+    _dtype = None
+    _min_value = None
+    _max_value = None
+    _rounding_digits = None
+
+    def __init__(self, learn_rounding_scheme=False, enforce_min_max_values=False,
```

Comment:
SDV sets `learn_rounding_scheme` and `enforce_min_max_values` to True by default, while RDT uses False. I'm not sure why the defaults are different, so I used RDT's since then I didn't have to change the tests.

It doesn't matter much either way, since I don't rely on the default values anywhere.